### ISSUE TITLE
How to use the visualization for custom BERT

### ISSUE THREAD

**Issue by QinghangHong1** [jessevig/bertviz#59]
Hi, I have a model, in which I have a fully connected layer after BERT. How do I visualize the attention weights with this model where the BERT output is an intermediate result?

My foward function of the model looks like 
`   _, pooled_output = self.bert(tokens, attention_mask=masks, output_all_encoded_layers=False)`
` x = self.Linear(pooled_output)`
`return x`
Where self.Linear is a fully connected layer.
Of course I need some activation functions and other elements, but I just try to simplified things.
Also, I am using package `pytorch_pretrained_bert`
Thank you a lot!

**Comment by jessevig** [comment]
First you would need to be sure what when you initialize the bert model, you set `output_attentions = True`.  Second, you would also need to return the attention data returned from calling the bert model: `attention = model(input_ids)[-1]` instead of just returning x. Let me know if this makes sense.
